MAINTAINERS.md

```diff
@@ -2,12 +2,27 @@
 
 ## Cutting a New Release
 
+> [!CAUTION]
+> Cutting a release, end-to-end, takes a reasonable amount of time --
+> mostly waiting on the CI -- and also requires the availability of
+> colleagues to properly review and approve.
+>
+> **Plan for it to take an entire working day.**
+>
+> Do not rush the process and avoid days on which you and colleagues
+> have limited availability. Don't release on the last working day
+> before a weekend or holiday, etc. If something goes awry before the
+> point of no return, keep calm and _don't_ carry on: Better to revert
+> and try again the next day than having a broken release (and a
+> stressful evening)!
+
 ### 1. Create a release preparation branch from `main`
 
 If necessary, cherry-pick the commits that should be included in the
 release. (This should only be done in exceptional circumstances.)
 
-- Update the [`CHANGELOG`], if necessary.
+- Update the [`CHANGELOG`], if necessary. This is used in the release
+  announcement, so ensure it conforms to the correct format.
```

Comment:
What is the correct format, and where can one find it?

Reply:
Good call: It's described in the `CHANGELOG` itself, but it's worth mentioning here as well :+1: